db/app.js

```diff
@@ -1,9 +1,14 @@
 const express = require("express");
 const app = express();
-const { getTopics, getApiEndpoints } = require("./controllers/controller");
+const {
+  getTopics,
+  getApiEndpoints,
+  getArticleByID,
+} = require("./controllers/controller");
 
 app.get("/api/topics", getTopics);
 app.get("/api", getApiEndpoints);
+app.get("/api/articles/:article_id", getArticleByID);
 
 app.use((req, res, next) => {
   res.status(404).send({ error: "Not Found" });
```

Comment:
Your 'Path not found' handling here will fall into issues as soon as your error handling middleware needs to handle other errors (like it will for this endpoint). The default error handling in this middleware can't be a 404 anyway because it has to be the 500 internal server error. This means we need to handle paths that don't exist in our server somewhere else in the app.js file.

Please have a look on the internet and see how people use `app.all` to catch any endpoints that don't exist in the server endpoints and refactor this 404